### PR TITLE
Question and instructions page

### PR DESCRIPTION
**Pull request by darshandamre**
Added Instructions Page,
Questions page
[Preview deploy](https://quiz-app-git-question-and-instructions-page-darshandamre.vercel.app/)

### CODE REVIEW

**Review by shraddha15-creator — APPROVED** [review]
Nice UI
Code looks clean, readable

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean with proper naming with good UI

File: src/components/Card.jsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-      </div>
-      {/* </a> */}
+      <Link to={`/instructions?quiz=${quiz.id}`}>
+        <img className="card__image" src={quiz.thumbnail} alt="Tenet" />
+        <div className="card__content">
+          <h3 className="h3 m-3">{quiz.title}</h3>
+          <p className="text-base ta-start m-3">
+            Take the quiz and find out.
+            <br />
+            {quiz.totalQuestions} questions
```
Comment (Shailseen) [inline src/components/Card.jsx:13]: you can use object destructuring here.

**Review by Shailseen — COMMENTED** [review]
Clean UI, nice folder structure. Good to merge.

**Pull request merged**